platform/darwin/src/MGLGeometry.h

```diff
@@ -45,6 +45,23 @@ typedef struct MGLCoordinateBounds {
     CLLocationCoordinate2D ne;
 } MGLCoordinateBounds;
 
+/** 
+ A quadrilateral area as measured on a two-dimensional map projection.
+ `MGLCoordinateQuad` differs from `MGLCoordinateBounds` in that it allows
+ representation of non-axis aligned bounds and non-rectangular quadrilaterals
+ */
+typedef struct MGLCoordinateQuad {
+    /** Coordinate at the top left corner. */
+    CLLocationCoordinate2D topLeft;
+    /** Coordinate at the top right corner. */
+    CLLocationCoordinate2D topRight;
+    /** Coordinate at the bottom right corner. */
+    CLLocationCoordinate2D bottomRight;
+    /** Coordinate at the bottom left corner. */
+    CLLocationCoordinate2D bottomLeft;
```

Comment:
The order in which these fields are declared affects the order of the Swift initializer’s arguments. It also affects how the undesignated initializer syntax in Objective-C (`{1, 2, 3, 4}`) is interpreted.